employees/serializers.py

```diff
@@ -50,3 +50,10 @@ class Meta:
             'description',
             'work_hours',
         )
+
+    def to_representation(self, instance):
+        data = super(ReportSerializer, self).to_representation(instance)
+        # SELF NOTE: For some reason, tests for list get methods got a Queryset as instance in serializer and I dunno why...
```

Comment:
Add it as a issue and in have to be investigate later. After adding issue please delete comment.

Reply:
![image](https://user-images.githubusercontent.com/17569113/52638911-53c66980-2ed3-11e9-93d0-e9f2b1293172.png)

Reply:
Done